source/preprocessor.tex

```diff
@@ -739,7 +742,8 @@
 \end{itemize}
 
 \pnum
-A macro directive is \defnx{active}{macro definition!active} at a source location
+\indextext{active macro directive|see{macro, active}}%
```

Comment:
Not sure this index is needed.

Reply:
Well, we do define "active macro directive" in this sentence, so it seems plausible to have an index entry for that exact phrase, although we'd like people to look up the meat under "macro".